### PR TITLE
Remove PlanPointer

### PR DESCRIPTION
**Pull request by mari-linhares**

### CODE REVIEW

File: syft/workers/base.py [hunk 1]
```diff
@@ -739,29 +741,29 @@ def search(self, *query):
...
 
-                if tensor.tags is not None:
-                    if query_item in tensor.tags:
+                if obj.tags:
+                    if query_item in obj.tags:
                         match = True
 
-                if tensor.description is not None:
-                    if query_item in tensor.description:
+                if obj.description:
```
Comment (robert-wagner) [inline syft/workers/base.py]: If the description is 0 or an empty string this will return incorrectly. The previous check was correct

**Comment by LaRiffle** [comment]
Except the point you mention everything is good for me!

**Comment by LaRiffle** [comment]
@mari-linhares I'll add an option to have several locations at the same time - it's work I should have cleaned and pushed, so if you need it tell me I will make it available faster

File: syft/workers/base.py (hunk 1 above)
Comment (mari-linhares) [reply syft/workers/base.py]: Thanks for the careful review! Done.

**Pull request merged**